tests/system/test_system.py

```diff
@@ -142,32 +145,121 @@ def tearDown(self):
     def _logger_name(prefix):
         return prefix + unique_resource_id("-")
 
-    def test_list_entry_with_unregistered(self):
-        from google.protobuf import any_pb2
+    @staticmethod
+    def _to_value(data):
+        if data is None:
+            return Value(null_value=NullValue.NULL_VALUE)
+        elif isinstance(data, numbers.Number):
+            return Value(number_value=data)
+        elif isinstance(data, str):
+            return Value(string_value=data)
+        elif isinstance(data, bool):
+            return Value(bool_value=data)
+        elif isinstance(data, (list, tuple, set)):
+            return Value(
+                list_value=ListValue(values=(TestLogging._to_value(e) for e in data))
+            )
+        elif isinstance(data, dict):
+            return Value(struct_value=TestLogging._dict_to_struct(data))
+        else:
+            raise TypeError("Unknown data type: %r" % type(data))
+
+    @staticmethod
+    def _dict_to_struct(data):
+        return Struct(fields={k: TestLogging._to_value(v) for k, v in data.items()})
+
+    def test_list_entry_with_auditlog(self):
```

Comment:
worth adding a comment describing the test case. Current name is too generic to explain what is specifically done in the test with audit logs